includes/admin/assets.php

```diff
@@ -129,5 +141,10 @@ public function load()
 			wp_enqueue_style('sc-welcome-style', $css_path . 'sc-welcome-pg-style.min.css', [], SIMPLE_CALENDAR_VERSION);
 			wp_enqueue_style('sc-tail-style', $css_path . 'tailwind.min.css', [], SIMPLE_CALENDAR_VERSION);
 		}
+		$run_oauth_helper = get_option('simple_calendar_run_oauth_helper');
+		if($run_oauth_helper){
+			wp_enqueue_style('sc-oauth-helper-style', $css_path . 'oauth-helper-admin.min.css', [], SIMPLE_CALENDAR_VERSION);
```

Comment:
Same here, shoudn't this be loaded only on settings page?

Reply:
Added condition for this also.